content/en/tracing/troubleshooting/_index.md

```diff
@@ -124,6 +124,34 @@ kubectl exec -it <agent-pod-name> -c trace-agent -- agent flare <case-id> --loca
 
     Knowing what versions are being used allows us to ensure integrations are supported in our [Compatiblity Requirements][15] section, check for known issues, or to recommend a tracer or language version upgrade if it will address the problem.
 
+## Service naming convention issues
+
+If you run into quota-related issues, try following the steps below to ensure you're following best practices for service naming conventions.
+
+### Exclude environment tag values from service names
+
+By default, the environment (`env`) is the primary tag for Datadog APM.
```

Comment:
May be worth linking to this APM guide here: https://docs.datadoghq.com/tracing/guide/setting_primary_tags_to_scope/.